cmd/avd_generator/main.go

```diff
@@ -72,3 +104,37 @@ var docsMarkdownTemplate = `
 {{ end}}
 {{ end }}
 `
+
+var terraformMarkdownTemplate = `
+{{ .Resolution }}
+
+{{ if .Terraform.GoodExamples }}
+{{ range .Terraform.GoodExamples }}
+{{ "```hcl" }}
```

Comment:
This doesn't work. I can't figure out how to escape the backticks in the template.